samples/msgext-northwind-inventory-python/lab/Exercise 01 - Set up.md

```diff
@@ -0,0 +1,118 @@
+
+> NOTE: Do not install the pre-release version as it hasn't been tested with this lab
+
+Open your working folder in Visual Studio Code. You may be asked to trust the authors of this folder; if so, please do.
+
+In the left sidebar, select the Extensions icon1️⃣. Enter the word "teams" into the search box 2️⃣ and locate "Teams Toolkit" in the search results. Click "Install" 3️⃣.
+
+![Installing Teams Toolkit in Visual Studio Code](./images/01-04-Install-TTK-01.png)
+
+Now select the Teams Toolkit icon in the left 1️⃣ . If it offers options to create a new project, you're probably in the wrong folder. In the Visual Studio Code file menu select "Open Folder" and directly open the **msgext-northwind-inventory-python** folder. You should see sections for Accounts, Environment, etc. as shown below.
+
+Similarly, search for "Python" in the search bar and locate it in the search results. Then, click "Install".
+
+![Installing python in Visual Studio Code](./images/01-04-Install-python-01.png)
+
+Under "Accounts" click "Sign in to Microsoft 365"2️⃣ and log in with your own Microsoft 365 account. You can get a free Microsoft 365 subscription for development purposes by joining the [Microsoft 365 Developer Program](https://developer.microsoft.com/microsoft-365/dev-program).
+
+> [!NOTE]
+> The Microsoft 365 Developer Program doesn't include Copilot for Microsoft 365 licenses. As such, if you decide to use a developer tenant, you will be able to test the sample only as a Message Extension.
+
+![Logging into Microsoft 365 from within Teams Toolkit](./images/01-04-Setup-TTK-01.png)
+
+A browser window will pop up and offer to log into Microsoft 365. When it says "You are signed in now and close this page", please do so.
+
+Now verify that the "Sideloading enabled" checker has a green checkmark. If it doesn't, that means your user account doesn't have permission to upload Teams applications. This permission is "off" by default; here are [instructions for enabling users to upload custom apps](https://learn.microsoft.com/microsoftteams/teams-custom-app-policies-and-settings#allow-users-to-upload-custom-apps)
+
+![Checking that sideloading is enabled](./images/01-04-Setup-TTK-03.png)
+
+Additionally, ensure that you have installed the Azure Account extension in Visual Studio Code and have logged in with an account that has access to an active Azure subscription. If you have not already signed in, you may be prompted to do so.
+
+Next, set up a resource group to provision resources to the connected Azure subscription:
+
+1. Open the **Azure Portal** and sign in with your Azure account.
+2. Go to **Resource Groups** and click on **+ Create**.
+3. Fill in the required details such as **Subscription**, **Resource Group Name**, and **Region**.
+4. Click **Review + Create**, then **Create** to finalize the resource group creation.
+
+This resource group will be used to provision resources using Azure Resource Manager (ARM) templates.
```

Comment:
If you create the resource group manually in the Azure Portal then you also need to update the environment variables used by the `arm/deploy` task. 

However, these manual steps are not required and can be removed. 

When the developer runs the app using F5 for the first time, Teams Toolkit will ask the developer to sign in to an account that has access to an Azure subscription (if not already logged in), select a subscription and either create a new or select an existing resource group in the selected subscription.

The steps that the developer needs to take to select the subscription and create the resource group should be included in exercise two.

Reply:
Fixed